### ISSUE TITLE
Broken with HA 0.96

### ISSUE THREAD

**Issue by BudBundi** [liudger/BSB-LAN-Component-for-Home-Assistant#5]
Climate backend has changed see https://www.home-assistant.io/blog/2019/07/17/release-96/ and https://developers.home-assistant.io/blog/2019/07/03/climate-cleanup.html

**Comment by liudger** [comment]
Yes I will update the code soon. I just need some time. Thanks for the links!
